src/Mod/TechDraw/TDTest/TDPyTest.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
```

Comment:
This does not have anything to do with the moving, but I believe these lines aren't needed when it's run through FreeCAD. From some very quick searches online, they are used to make sure that they are interpreted by Python if opened isolated on Linux.

It was just a very quick serach, but I don't think they are needed

Reply:
do these lines hurt anything?

Reply:
Not at all